### PR TITLE
Html in the Wizard (for string fields)

### PR DESCRIPTION
**Pull request by kuzmadom**

### CODE REVIEW

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10725-1868/wizard-1868/ru/">wizard (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10725-1868/wizard-1868/en/">wizard (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10725-1868/wizard-1868/ru/">wizard (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-10725-1868/wizard-1868/en/">wizard (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
Please merge the Pull Request with the changes from Weblate into your branch.

**Review by korvin89 — APPROVED** [review]

File: src/ui/libs/DatalensChartkit/modules/html-generator/index.ts [hunk 1]
```diff
@@ -38,9 +39,17 @@ export function generateHtml(
         const {tag, attributes = {}, style = {}, content, theme} = item;
 
         if (!ALLOWED_TAGS.includes(tag)) {
-            throw new ChartKitCustomError(null, {
-                details: `Tag '${tag}' is not allowed`,
+            const msg = `Tag '${tag}' is not allowed`;
```
[the remark this answers is not in the document]
Comment (kuzmadom) [reply src/ui/libs/DatalensChartkit/modules/html-generator/index.ts:42]: different return values from a potential function - therefore it was difficult to separate into a single piece

**Pull request merged**